lib/Transforms/Scalar/DxilRemoveDeadBlocks.cpp

```diff
@@ -175,6 +274,119 @@ static bool EraseDeadBlocks(Function &F, DxilValueCache *DVC) {
   return true;
 }
 
+static bool DeleteDeadBlocks(Function &F, DxilValueCache *DVC) {
+  DeadBlockDeleter Deleter;
+  bool Changed = false;
+  constexpr unsigned MaxIteration = 10;
+  for (unsigned i = 0; i < MaxIteration; i++) {
+    bool LocalChanged = Deleter.Run(F, DVC);
+    Changed |= LocalChanged;
+    if (!LocalChanged)
+      break;
+  }
+  return Changed;
+}
+
+static bool IsDxBreak(Instruction *I) {
+  CallInst *CI = dyn_cast<CallInst>(I);
+  if (!CI) return false;
+  Function *CalledFunction = CI->getCalledFunction();
+  return CalledFunction && CalledFunction->getName() == hlsl::DXIL::kDxBreakFuncName;
+}
+
+static bool IsIsHelperLane(Instruction *I) {
```

Comment:
You can use `IsDxilOpFuncCallInst(I, hlsl::DXIL::OpCode::IsHelperLane)` instead.